doc/en/user/source/community/ogc-api/features/index.rst

```diff
@@ -0,0 +1,202 @@
+OGC API Features
+================
+
+An `OGC Features API <https://github.com/opengeospatial/ogcapi-features>`_ up to date with the  released version of the Features API, along with implementation of the `draft CQL filtering extension <https://github.com/opengeospatial/ogcapi-features/tree/master/extensions/cql>`_ and the `draft multi-crs extension <https://github.com/opengeospatial/ogcapi-features/tree/master/extensions/crs>`__.
```

Comment:
CRS is actually [a published standard](https://www.ogc.org/standards/ogcapi-features) now, but we have not aligned to its latest version (not sure if there is any alignement to make, to start with, TBH):

![image](https://user-images.githubusercontent.com/325433/108349703-9b2f1100-71e3-11eb-9742-4899df17e996.png)

The CQL extension is in "public comment" stage, same deal, not sure if we need to change anything to comply (probably we'll have to make some light changes to the CQL parser).

Reply:
Thanks for the update, captured in docs as a table which should be easier to maintain
![image](https://user-images.githubusercontent.com/629681/108379694-09131280-71bb-11eb-8d72-e52e973e1009.png)